packages/ocaml-options-vanilla/ocaml-options-vanilla.1/opam

```diff
@@ -9,6 +9,11 @@ conflicts: [
   "ocaml-option-32bit"
   "ocaml-option-afl"
   "ocaml-option-bytecode-only"
+    {ocaml-system:version < "5" |
+     ocaml-base-compiler:version < "5" |
+     ocaml-variants:version < "5" |
+     arch = "arm64" |
+     arch = "x86_64" }
```

Comment:
Doesn't that mean that every new supported architecture on OCaml 5 will trigger such a rebuild ?

Reply:
yes, any change in the `depends` field with opam < 2.2 triggers a rebuild

Reply:
ah it looks like since these are `post` dependencies it might be fine in the end. From quick testing:
* opam 2.0 does not do anything
* opam 2.1 does reinstall just ocaml-options-vanilla
* opam 2.2 does nothing

But i suppose it requires more testing to be sure